app/controllers/Application.scala

```diff
@@ -46,24 +47,28 @@ class Application(
   }
 
   def bundleLanding(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
-    if (newDesigns == "circles") {
-      Ok(views.html.bundleLanding(
-        title,
-        "support-landing-page-old",
-        "supportLandingPageOld.js",
-        contributionsPayPalEndpoint,
-        description = Some(stringsConfig.bundleLandingDescription)
-      ))
-    } else if (newDesigns == "circles-garnett") {
-      Ok(views.html.bundleLanding(
-        title,
-        "support-landing-page",
-        "supportLandingPage.js",
-        contributionsPayPalEndpoint,
-        description = Some(stringsConfig.bundleLandingDescription)
-      ))
+    if (unsupportedBrowser) {
```

Comment:
Is the US landing page something to look at in the future once we're confident this works for the UK one?

Reply:
Yup. Potentially every page - in case someone arrives directly from elsewhere without passing through the landing page.